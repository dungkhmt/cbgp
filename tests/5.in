20 20
20 30
12 5
10 0
7 2
17 10
8 5
15 11
15 0
17 9
19 7
16 5
8 2
7 18
4 9
7 1
9 14
18 0
13 0
6 2
7 3
14 17
18 1
11 10
12 4
5 2
2 0
13 4
14 11
12 13
5 13
18 2
1
0 0 21 1 20 3 30 0 14 6 16 4 21 4 27 0 18 5 13 7 9 7 11 9 15 5 12 4 12 8 10 10 21 9 11 7 20 0 23 4 
