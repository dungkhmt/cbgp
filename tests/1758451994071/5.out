(12, 5), (10, 0), (7, 2), (17, 10), (8, 5), (15, 11), (15, 0), (17, 9), (19, 7), (16, 5), (8, 2), (7, 18), (4, 9), (7, 1), (9, 14), (18, 0), (13, 0), (6, 2), (7, 3), (14, 17), (18, 1), (11, 10), (12, 4), (5, 2), (2, 0), (13, 4), (14, 11), (12, 13), (5, 13), (18, 2), 
numberIntersection=0 
minAngle=0.5338445800726763 
sumAngle=18.685141748496306 
minEdgeLength=1.0 
sumEdgeLength=597.5175971465111 
minDistance=0.5825189414280086 
sumDistance=15673.295203389402 

0.0000000000 0.5338445801 18.6851417485 1.0000000000 597.5175971465 0.5825189414 15673.2952033894
0: (63, 62),
1: (73, 26),
2: (65, 41),
3: (3, 0),
4: (52, 48),
5: (62, 43),
6: (16, 9),
7: (58, 25),
8: (62, 41),
9: (47, 39),
10: (42, 50),
11: (19, 62),
12: (56, 45),
13: (59, 54),
14: (26, 42),
15: (23, 63),
16: (2, 0),
17: (42, 49),
18: (79, 40),
19: (80, 0),
63 62 73 26 65 41 3 0 52 48 62 43 16 9 58 25 62 41 47 39 42 50 19 62 56 45 59 54 26 42 23 63 2 0 42 49 79 40 80 0 
0: (63, 62), 1: (73, 26), 2: (65, 41), 3: (3, 0), 4: (52, 48), 5: (62, 43), 6: (16, 9), 7: (58, 25), 8: (62, 41), 9: (47, 39), 10: (42, 50), 11: (19, 62), 12: (56, 45), 13: (59, 54), 14: (26, 42), 15: (23, 63), 16: (2, 0), 17: (42, 49), 18: (79, 40), 19: (80, 0), 
numberIntersection=0 minAngle=0.5338445800726763 sumAngle=18.68514174844602 minEdgeLength=1.0 sumEdgeLength=597.5175971465321 minDistance=0.5825189414280086 sumDistance=15673.295203388348 
Time taken: 13.266 s
